dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210224DropUserProxyTable.java

```diff
@@ -0,0 +1,47 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+/**
+ * Task used to remove user_proxy table from database
+ */
+public class Task210224DropUserProxyTable implements StartupTask {
+
+    @Override
+    public void executeUpgrade() throws DotDataException, DotRuntimeException {
+
+        DotConnect dotConnect = new DotConnect();
```

Comment:
I think we have the:

```
DotDatabaseMetaData.tableExists method for this propose
```

Reply:
I just see you have to call on the force.
I do not understand why do you need to double check on oracle

Reply:
I didn't know we had `DotDatabaseMetaData .tableExists`, I will update it. 

The reason why I modified the logic for Oracle was that this DB is case sensitive, so `USER_PROXY` is not the same as `user_proxy`. This validation won't be necessary anymore because the  `DotDatabaseMetaData .tableExists` already considers it.